src/forms/FormFieldsContainer.js

```diff
@@ -256,16 +256,20 @@ const styles = {
     paddingLeft: 5
   },
   formTitles: {
+    fontFamily: 'Roboto',
     borderRadius: 4,
-    border: '1px solid ' + settings.mediumGrey,
-    width: '75%',
+    border: '1px solid #d8d8d8',
```

Comment:
No reason behind that. I completely forgot about the settings file. I added custom settings only for buttons. I'll add that back